commonui/src/main/java/com/hyperwallet/android/ui/common/util/CurrencyParser.java

```diff
@@ -127,4 +127,22 @@ public CurrencyDetails getCurrency(String currencyCode) {
         }
         return null;
     }
+
+    public static String getRateWithFourDecimal(String rate) {
+        if (rate != null) {
+            StringBuilder builder = new StringBuilder();
+            String[] amount = rate.split("\\.");
```

Comment:
why use string manipulation? Parsing to double isn't working?

Reply:
NumberFormat/Decimal formate rounding the value 1.0000 as 1 , i.e used string manipulation iOS also implemented the same.

Reply:
please re-read my comment. Why can't we parse value to double and then truncate decimals ??
![Screen Shot 2021-03-24 at 7 00 33 PM](https://user-images.githubusercontent.com/37385596/112407385-3c255600-8cd3-11eb-8431-62cacfff6217.png)

Reply:
once we parse the value to double and truncate , its formatted and rounding the value eg: 1.000056 its return as 1. not as 1.0000

Reply:
Brandon confirm implement the round up for FX as a  new ticket for IOS and android